recipe/meta.yaml

```diff
@@ -13,28 +13,31 @@ source:
       - CMakeLists.txt.diff
 
 build:
-  number: 3
+  number: 1003
   # This package is a dependency of hdfs3, which is primarily used with HDFS on Linux.
   skip: True  # [win]
 
 requirements:
   build:
     - cmake
-    - boost-cpp 1.66.0
+    - "{{ compiler('c') }}"
+    - "{{ compiler('cxx') }}"
+  host:
+    - boost-cpp
     - libgsasl 1.8.*
     - libntlm
-    - libprotobuf 3.5.*
+    - libprotobuf
     - libuuid
-    - libxml2 2.9.*
-    - krb5 1.14.*
+    - libxml2
+    - krb5
   run:
-    - boost-cpp 1.66.0
+    - boost-cpp
     - libgsasl 1.8.*
```

Comment:
Would be nice if we could move this pinning out of the recipe as well either via the global pinning file or run exports.